models/demos/convnet_mnist/tt/convnet_mnist.py

```diff
@@ -35,13 +35,41 @@ def convnet_mnist(
         packer_l1_acc=False,
     )
     x = ttnn.to_layout(input_tensor, layout=ttnn.ROW_MAJOR_LAYOUT)
+
+    tt_weight = parameters.conv1.weight
+    tt_bias = parameters.conv1.bias
+    conv_kwargs = {
+        "input_layout": x.get_layout(),
+        "in_channels": 1,
+        "out_channels": 32,
+        "batch_size": batch_size,
+        "input_height": input_tensor.shape[1],
+        "input_width": input_tensor.shape[2],
+        "kernel_size": (3, 3),
```

Comment:
seems like most args are redundant between prepare_conv_weights and conv2d
I don't this we should hardcode params twice like kernel size (3, 3), all of these should be defined in one place.

It's weird that for prepare_conv_weights we use this kwargs dict, and on conv2d we use named argumets, I would align those.

Also, there is not need to define this conv_kwargs in case it's not going to be used, it should be under if where ttnn.prepare_conv_weights is.

This comment applies to all models changed.